packages/react-native/Libraries/Image/RCTImageUtils.mm

```diff
@@ -55,6 +55,68 @@ static CGImagePropertyOrientation CGImagePropertyOrientationFromUIImageOrientati
   }
 }
 
+static UIImage* decodeImageFromCGImageSourceRef(
+    CGImageSourceRef sourceRef,
+    CGSize destSize,
+    CGFloat destScale,
+    RCTResizeMode resizeMode)
+{
+  if (!sourceRef) {
```

Comment:
can you link the source from where you copied this, please? It will make it easier to review the code. Perhaps there is a way to do it with just reusing the other code rather then reimplementing it! 😉

Reply:
sure @cipolleschi 
https://github.com/facebook/react-native/blob/main/packages/react-native/Libraries/Image/RCTImageUtils.mm

check from line 260